ExomeDepth/make_BEDdetail.py

```diff
@@ -118,6 +117,54 @@ def slice_vcf(args, merge_dic):
     excluded_samples_file.close()
     return event_dic, children, parents
 
+def calculate_statistics(data, status):
+    dic = {}
```

Comment:
dic hernoemen naar stats_dic of gewoon stats.

En je kan ook dit moeteen doen:
```python
stats = {
    'cr_avr_{0}'.format(status) : "n/a"
}

```

en dan in totalcount if:
```python
stats[cr_avr_{0}'.format(status)] = "%.3f" % (statistics.mean(data['correlation']))
```